ruma-events/src/room/avatar.rs

```diff
@@ -43,3 +44,40 @@ impl AvatarEventContent {
         Self::default()
     }
 }
+
+/// Metadata about an image (specific to avatars).
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct ImageInfo {
+    /// The height of the image in pixels.
+    #[serde(rename = "h", skip_serializing_if = "Option::is_none")]
+    pub height: Option<UInt>,
+
+    /// The width of the image in pixels.
+    #[serde(rename = "w", skip_serializing_if = "Option::is_none")]
+    pub width: Option<UInt>,
+
+    /// The MIME type of the image, e.g. "image/png."
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub mimetype: Option<String>,
+
+    /// The file size of the image in bytes.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub size: Option<UInt>,
+
+    /// Metadata about the image referred to in `thumbnail_url`.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub thumbnail_info: Option<Box<ThumbnailInfo>>,
+
+    /// The URL to the thumbnail of the image. Only present if the thumbnail is unencrypted.
```

Comment:
```suggestion
    /// The URL to the thumbnail of the image.
```

Since there is no possibility of encryption here, we should remove this part of the documentation.

Reply:
Removed.